src/chart/context.rs

```diff
@@ -507,6 +505,9 @@ impl<'a, DB: DrawingBackend, X: Ranged, Y: Ranged> ChartContext<'a, DB, RangedCo
                 tick_size.abs() * 2
             };
 
+        /* All labels are right-aligned. */
+        let label_style = &label_style.alignment(TextAlignment::Right);
```

Comment:
And I believe `DrawingArea::titled` is also benefited from this. (Use middle alignment)

Reply:
Sure, I'll fix it.